pkg/store/kotsstore/kots_store.go

```diff
@@ -194,3 +197,53 @@ func (c KOTSStore) GetClientset() (*kubernetes.Clientset, error) {
 
 	return clientset, nil
 }
+
+func (s KOTSStore) getConfigmap(name string) (*corev1.ConfigMap, error) {
+	clientset, err := s.GetClientset()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to get clientset")
+	}
+
+	existingConfigmap, err := clientset.CoreV1().ConfigMaps(os.Getenv("POD_NAMESPACE")).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil && !kuberneteserrors.IsNotFound(err) {
+		return nil, errors.Wrap(err, "failed to get configmap")
+	} else if kuberneteserrors.IsNotFound(err) {
+		configmap := corev1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: "v1",
+				Kind:       "ConfigMap",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: os.Getenv("POD_NAMESPACE"),
+				Labels: map[string]string{
+					"owner": "kotsadm",
```

Comment:
This doesn't look like a real label.  Everything else uses [these](https://github.com/replicatedhq/kots/blob/3f5e3166c8bb14666ba696eaac6ebbefc4310f6f/pkg/kotsadm/types/constants.go#L16)